src/ui/components/shared/FirstReplayModal/FirstReplayModal.tsx

```diff
@@ -47,12 +45,10 @@ function FirstReplayModal({ hideModal }: PropsFromRedux) {
           </OnboardingBody>
         </OnboardingContent>
         <OnboardingActions>
-          <PrimaryLgButton color="blue" onClick={handleOpen}>
-            {`Ready as I'll ever be, Doc`}
-          </PrimaryLgButton>
-          <PrimaryLgButton color="gray" onClick={handleSkip}>
```

Comment:
What was "gray" primary button supposed to be? Gray is the color we use for _disabled_ buttons, so I've replaced this with the "outline" variant.